Package.swift

```diff
@@ -1,5 +1,15 @@
-// swift-tools-version:5.0
-
+// swift-tools-version:5.1
 import PackageDescription
 
-let package = Package(name: "Dollar")
+let package = Package(
+    name: "Dollar",
+    platforms: [
+        .iOS(.v9),
```

Comment:
Can you add other platforms as well like macOS, watchOS, tvOS, and linux.

Reply:
good call. i will do this.